### PR TITLE
Upgrade to springboot 3.2.3 and dubbo 3.3.0-beta.1

### PR DESCRIPTION
**Pull request by kwings6**
Upgrade to springboot3.2.3 , dubbo 3.3.0-beta.1 and nacos 2.3.1. Modify the project into three modules. Fixed the bug which lead to dubbo cache conflict in local machine.

### CODE REVIEW

File: 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-provider/src/main/java/org/apache/dubbo/samples/registry/ProviderApplication.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import org.apache.dubbo.config.spring.context.annotation.EnableDubbo;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+@EnableDubbo
+public class ProviderApplication {
+    public static void main(String[] args) {
+        SpringApplication.run(ProviderApplication.class, args);
```
Comment (chickenlj) [inline 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-provider/src/main/java/org/apache/dubbo/samples/registry/ProviderApplication.java:27]: Please add `System.out.println("dubbo service started");` here for it's a checking point for the testing framework https://github.com/apache/dubbo-samples/pull/1086/files#diff-162f44ea31c4096aa09a7da9b61f87efeded56aefa1811e8d9a4a1be91bf05f8R41

File: 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-consumer/src/main/java/org/apache/dubbo/samples/registry/consumer/ConsumerApplication.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+import java.io.File;
+
+@SpringBootApplication
+@EnableDubbo
+public class ConsumerApplication {
+    public static void main(String[] args) {
+        SpringApplication application = new SpringApplication(ConsumerApplication.class);
+
+        //修改dubbo的本地缓存路径，避免本地启动多个应用导致缓存冲突
```
Comment (chickenlj) [inline 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-consumer/src/main/java/org/apache/dubbo/samples/registry/consumer/ConsumerApplication.java:32]: Please translate to English

**Review by chickenlj — CHANGES_REQUESTED** [review]

File: 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-consumer/src/main/resources/application.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dubbo:
+    application:
+        name: dubbo-samples-nacos-registry-consumer
+        qos-port: 20994
+    protocol:
+        name: dubbo
+        port: 20884
```
Comment (chickenlj) [inline 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-consumer/src/main/resources/application.yml:23]: consume does not need `protocol` configuration, please remove it.

File: 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-provider/src/main/resources/application.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dubbo:
+   application:
+        name: dubbo-samples-nacos-registry-consumer
+        qos-port: 20993
+   protocol:
+        name: dubbo
+        port: 20883
```
Comment (chickenlj) [inline 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-provider/src/main/resources/application.yml:23]: Make sure this port is the same as the value here  https://github.com/apache/dubbo-samples/pull/1086/files#diff-162f44ea31c4096aa09a7da9b61f87efeded56aefa1811e8d9a4a1be91bf05f8R40

Please use `20880`

File: 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/pom.xml [hunk 5]
```diff
@@ -96,69 +106,25 @@
                 </exclusion>
             </exclusions>
         </dependency>
```
Comment (chickenlj) [inline 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/pom.xml:108]: please remove `dubbo-configcenter-nacos` and `dubbo-registry-nacos` dependency

File: 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-provider/pom.xml [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-log4j2</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.alibaba.nacos</groupId>
+            <artifactId>nacos-client</artifactId>
+            <version>2.3.1</version>
+        </dependency>
+    </dependencies>
```
Comment (chickenlj) [inline 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-provider/pom.xml:59]: Please use `dubbo-nacos-spring-boot-starter` instead.

File: 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-consumer/pom.xml [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-log4j2</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.alibaba.nacos</groupId>
+            <artifactId>nacos-client</artifactId>
+            <version>2.3.1</version>
+        </dependency>
+    </dependencies>
```
Comment (chickenlj) [inline 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-consumer/pom.xml:59]: Please use `dubbo-nacos-spring-boot-starter` instead.

**Review by chickenlj — CHANGES_REQUESTED** [review]

File: 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-consumer/pom.xml (hunk 7 above)
Comment (kwings6) [reply 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-consumer/pom.xml:59]: I solve the problems you mentioned but failing checks are still exist. The reason of failing checks is run test timeout .

Comment (kwings6) [reply 3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-consumer/pom.xml:59]: The reason for this failing checks are the same as in the past

**Pull request closed without merging**